### PR TITLE
Fixed SourceLevel js issues

### PR DESCRIPTION
**Pull request by camilorecce**
Fixed some SourceLevel JavaScript issues in app.js and socket.js, as #104

### CODE REVIEW

File: assets/js/socket.js
Comment (nelsonic) [reply assets/js/socket.js:60]:
```suggestion
  .receive('ok', resp => { console.log('Joined successfully', resp) });
```

File: assets/js/socket.js
Comment (nelsonic) [reply assets/js/socket.js:58]:
```suggestion
let channel = socket.channel('topic:subtopic', {});
```
Not that I necessarily _agree_ with using single quotes, but if SourceLevelBot wants it, it gets it!

File: assets/js/socket.js
Comment (nelsonic) [reply assets/js/socket.js:61]:
```suggestion
  .receive('error', resp => { console.log('Unable to join', resp) });
```

Comment (nelsonic) [reply assets/js/socket.js:61]:
```suggestion
  .receive('error', resp => { console.log('Unable to join', resp) });
```

**Comment by nelsonic** [comment]
The reason the Travis-CI build fails is because the branch (fork) is not owned by @dwyl so  the `AUTH_API_KEY` is unavailable: https://travis-ci.com/github/dwyl/auth/builds/188539795
image

**Review by nelsonic — APPROVED** [review]
@camilorecce thanks for these fixes. 🙌 
The only reason the Travis-CI build does not pass is because the branch is on a fork.
I have invited you to be a member of @dwyl and given you write access to this repo in case you want / have time to apply any more fixes. (no pressure) If you create a branch on _this_ repo Travis-CI will pass. 👍 

Thanks again. ☀️

**Comment by iteles** [comment]
Thank you @camilorecce, glad you found us! :blush:

**Comment by camilorecce** [comment]
Great, I will help with everything I can!
I am new to open source, so maybe I will ask a lot of questions :)
Is it safe to give me write access? I don't want to mess the code hahah
Thanks!

**Comment by nelsonic** [comment]
@camilorecce you can create branches and pull requests to your heart’s content. Don’t worry, you can’t mess anything because the master branch requires checks before it can be merged. 👍🏻
See: https://github.com/dwyl/protect-master-branch

**Comment by camilorecce** [comment]
I get a permission denied error on git when I try to create a branch. I can't create a branch on github neither. Some help? @nelsonic

**Comment by nelsonic** [comment]
@camilorecce please see: https://github.com/dwyl/auth/invitations
You need to accept the invite that was sent to you before you have access. 
github-auth-pending-invite

**Pull request merged**
